docs/sphinx/source/whatsnew/v0.9.0.rst

```diff
@@ -0,0 +1,66 @@
+.. _whatsnew_0900:
+
+v0.9.0 (MONTH DAY YEAR)
+-----------------------
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+
+Deprecations
+~~~~~~~~~~~~
+* The following ``ModelChain`` attributes are deprecated. They have been moved
+  to the :py:class:`~pvlib.modelchain.ModelChainResult` class that is
+  accessible via ``ModelChain.results``
+  * ``ModelChain.ac``
+  * ``ModelChain.airmass``
+  * ``ModelChain.aoi``
+  * ``ModelChain.aoi_modifier``
+  * ``ModelChain.cell_temperature``
+  * ``ModelChain.dc``
+  * ``ModelChain.diode_params``
+  * ``ModelChain.effective_irradiance``
+  * ``ModelChain.spectral_modifier``
+  * ``ModelChain.total_irrad``
+  * ``ModelChain.tracking``
+
+Enhancements
+~~~~~~~~~~~~
+* In :py:class:`~pvlib.modelchain.ModelChain`, attributes which contain
+  output of models are now collected into ``ModelChain.results``.
+  (:pull:`1076`, :issue:`1067`)
+* Added :py:class:`~pvlib.pvsystem.Array` class to represent an array of
+  modules separately from a :py:class:`~pvlib.pvsystem.PVSystem`.
+  (:pull:`1076`, :issue:`1067`)
+* Added capability for modeling a PV system with multiple arrays in
+  :py:class:`~pvlib.pvsystem.PVSystem`. Updates the ``PVSystem`` API
+  to operate on and return tuples where each element of the tuple corresponds
+  to the input or output for a specific ``Array``. (:pull:`1076`,
+  :issue:`1067`)
+* Support for systems with multiple ``Arrays`` added to
+  :py:class:`~pvlib.modelchain.ModelChain`. This includes substantial API
+  enhancements for accepting different weather input for each ``Array`` in the
+  system. (:pull:`1076`, :issue:`1067`)
+* Support for :py:func:`~pvlib.inverter.sandia_multi` added to
+  :py:class:`~pvlib.pvsystem.PVSystem` and
+  :py:class:`~pvlib.modelchain.ModelChain` (as ``ac_model='sandia_multi'``).
+  (:pull:`1076`, :issue:`1067`)
+
+Bug fixes
+~~~~~~~~~
+
+Testing
+~~~~~~~
+
+Documentation
+~~~~~~~~~~~~~
+
+Requirements
+~~~~~~~~~~~~
+* ``dataclasses`` is required for python 3.6
+
+Contributors
+~~~~~~~~~~~~
+* Will Holmgren (:ghuser:`wholmgren`)
+* Cliff Hansen (:ghuser:`cwhanse`)
+* Will Vining (:ghuser:`wfvining`)
```

Comment:
```suggestion
* Will Vining (:ghuser:`wfvining`)
* Anton Driesse (:ghuser:`adriesse`)
* Mark Mikofski (:ghuser:`mikofski`)
* Nate Croft (:ghuser:`ncroft-b4`)
* Kevin Anderson (:ghuser:`kanderso-nrel`)
```

Reply:
...folks that contributed to the conversation in the original issue and/or here in the pr, and/or reviewed the code.